tests/ci/autoscale_runners_lambda/test_autoscale.py

```diff
@@ -91,7 +91,14 @@ def test_normal_cases(self):
             ),
             TestCase("lower-min", 10, 5, 20, [Queue("queued", 5, "lower-min")], 10),
             # scale up group with prefix private-
-            TestCase("private-increase", 1, 13, 20, [Queue("queued", 10, "private-increase")], 10),
+            TestCase(
+                "private-increase",
+                1,
+                13,
+                20,
+                [Queue("queued", 10, "private-increase")],
+                10,
```

Comment:
It's called `increase`, but tests decrease. Another queue `in_progress` should do the trick

```suggestion
                [
                    Queue("in_progress", 12, "private-increase"),
                    Queue("queued", 11, "private-increase"),
                ],
                20,
```